src/oas/accessors.ts

```diff
@@ -1,12 +1,12 @@
-import { unionBy } from 'lodash';
+import { isObject, unionBy } from 'lodash';
 
 export function getOasParameters<ParamType extends { name: string; in: string }>(
```

Comment:
This should probably become `getValidOasParameters` – but we can keep it as it is.